system/settings/settings.xml

```diff
@@ -3111,7 +3111,7 @@
             <addontype>kodi.resource.language</addontype>
           </constraints>
           <control type="button" format="addon">
-            <show more="false" details="false">all</show>
+            <show more="true" details="true">installed</show>
```

Comment:
I'd keep "details" to "false" because we don't have proper addon icons for language addons.